integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -202,25 +210,37 @@ object BenchUtils {
         val end = System.nanoTime()
         val elapsed = NANOSECONDS.toMillis(end - start)
         queryTimes.append(elapsed)
-        println(s"$logPrefix Iteration $i took $elapsed msec.")
+
+        val status = if (taskFailureListener.taskFailures.isEmpty) {
+          STATUS_COMPLETED
+        } else {
+          // add the first task failure to the list of exceptions in the report
+          exceptions.append(taskFailureListener.taskFailures.head.toString)
+          STATUS_COMPLETED_WITH_TASK_FAILURES
+        }
```

Comment:
nit:
```
val failureOpt = taskFailureListener.taskFailures.headOption
val status = failureOpt.map(_ =>  STATUS_COMPLETED_WITH_TASK_FAILURES).getOrElse(STATUS_COMPLETED)
failureOpt.foreach(exceptions.add(_.toString))
```

Reply:
@gerashegalov I went ahead and implemented this suggestion since it is much cleaner. Could you re-approve, please.